src/lib/form-field/form-field.html

```diff
@@ -8,21 +8,10 @@
     <div class="mat-input-infix mat-form-field-infix" #inputContainer>
       <ng-content></ng-content>
 
-      <!--
-        TODO: remove `mat-input-placeholder-wrapper` and `mat-form-field-placeholder-wrapper`
-        next time we do breaking changes.
-        @deletion-target 6.0.0
-      -->
-      <span class="mat-form-field-label-wrapper mat-input-placeholder-wrapper mat-form-field-placeholder-wrapper">
+      <span class="mat-form-field-label-wrapper">
```

Comment:
Can you add a `@deletion-target` while you're here to remove all of the `mat-input-*` classes? We can do the actual remove in a different PR because I know for sure it's going to be a nightmare to merge

Reply:
Done. I've set it to 7.0.0, but I can bump it down to 6.0.0 if you want.